gallery/installing-psget.md

```diff
@@ -60,3 +64,23 @@ Exit
    Copy-Item "C:\LocalFolder\PowerShellGet\*" "$env:ProgramFiles\WindowsPowerShell\Modules\PowerShellGet\" -Recurse -Force
    Copy-Item "C:\LocalFolder\PackageManagement\*" "$env:ProgramFiles\WindowsPowerShell\Modules\PackageManagement\" -Recurse -Force
    ```
+
+#### Computers without PowerShellGet
+
+For a computer that doesn't have any version of **PowerShellGet** installed, you can download it from the [PowerShell Gallery](https://www.powershellgallery.com/).
```

Comment:
Let's rephrase this a bit.  That first sentence by itself makes it sound like a PS 3/4 computer without PowerShellGet can just get it from the Gallery.

Reply:
Thanks for the feedback. Fixed.